action/envelope.go

```diff
@@ -217,58 +218,12 @@ func (elp *envelope) Action() Action { return elp.payload }
 
 // ToEthTx converts to Ethereum tx
 func (elp *envelope) ToEthTx(evmNetworkID uint32, encoding iotextypes.Encoding) (*types.Transaction, error) {
-	tx, ok := elp.Action().(EthCompatibleAction)
+	act, ok := elp.Action().(EthCompatibleAction)
```

Comment:
still have `putpullresult` does not implement this, so for now this check is still needed
can remove this in another PR if we decide to implement `EthCompatibleAction` for `putpullresult`